control_panel_api/aws.py

```diff
@@ -0,0 +1,85 @@
+import json
+
+import boto3
+
+
+def create_bucket(name, **kwargs):
+    boto3.client('s3').create_bucket(
+        Bucket=name,
+        ACL=kwargs.get('acl', ''),
+        CreateBucketConfiguration={'LocationConstraint': kwargs.get('region', '')},
```

Comment:
If we switch `kwargs.get()` with `kwargs["ket"]` it would blow up right?

I think we should use (in general) a meaningful default for these options - one there is one - and just let it raise an Exception when a meaningful default doesn't exist and we want to make the argument mandatory.

Reply:
No should still work, but using `get` is the idiom and gives you a default.  Yep for stuff that can take a default and we have an option (like 'private') can use that.